implementation.js

```diff
@@ -2,8 +2,41 @@
 var ES = require('es-abstract/es6');
 var supportsDescriptors = require('define-properties').supportsDescriptors;
 
+var parseIterable = function (arrayLike) {
+	var done = false;
+	var iterableResponse;
+	var tempArray = [];
+
+	if (typeof arrayLike.next === 'function') {
+		while (!done) {
+			iterableResponse = arrayLike.next();
+			if (
+				iterableResponse.hasOwnProperty('value')
+				&& iterableResponse.hasOwnProperty('done')
+			) {
+				if (iterableResponse.done === true) {
```

Comment:
Can you add comments linking to the part of the spec where `done` must be a literal true, instead of just a truthy value, and so forth?